src/parser/druid/restoration/modules/core/Rejuvenation.js

```diff
@@ -2,7 +2,7 @@ import SPELLS from 'common/SPELLS';
 import Analyzer from 'parser/core/Analyzer';
 import HealingDone from 'parser/shared/modules/throughput/HealingDone';
 
-import Mastery from './/Mastery';
+import Mastery from "./Mastery";
```

Comment:
Intentional double quotes?

Reply:
The fixer for that fucked it up. I noticed but didn't care enough to do something about it.